docs/guide-for-local-setup/README.md

```diff
@@ -0,0 +1,38 @@
+# This is a document that describes how to run RDS app locally with local backend
+
+## 1. Setup the Backend
+
+- Go to the RDS Backend setup guide [here](https://github.com/Real-Dev-Squad/website-backend/#readme)
+- Add the `cors` config in `config/local.js` file:
+
+```javascript
+ cors: {
+     allowedOrigins: /(http:\/\/localhost:[0-9]{4}$)/,
+ },
+```
+
+## 2. Setup the Frontend -
+
+- Go to the README of the respective frontend repo.
+- Change all the API endpoints to `http://<your local api>:<port>/`
+  - RDS API endpoint: `https://api.realdevsquad.com/`
+  - Default API endpoint[local]: `http://localhost:3000/`
+- Change all the url with respective frontend url
+  - Like you want to run WWW and My site then you have to set `http://localhost:5500` for WWW(main Site) then change all the url from `https://www.realdevsquad.com` to `http://localhost:5500` in both the repo and if you set `http://localhost:3443` for My site then you have to change all the url from `https://my.realdevsquad.com` to `http://localhost:3443` in both the repo and so on.
+
+## 3. For skip the signup process
+
```

Comment:
How will they generate tokens or log in?
No steps for that

Reply:
> **Note:** For register your app for OAuth, go to url `http://<app url>/auth/github/callback` and authorize the app.

Last line of the doc

Reply:
I think it should be `/auth/github/login` can you please confirm once

Reply:
Done